.circleci/config.yml

```diff
@@ -95,6 +95,30 @@ commands:
           cd ~/repo/tests
           ./travis-run-tests.sh << parameters.flags >>
 
+  # Build and push PyNE website  
+  website_build_push:
+    description: "build PyNE website and push it"
+    parameters:
+      push_option:
+          type: string
+          default: "push-test"
+    steps:
+      - pull_container:
+          build: "python2_dagmc_pymoab" 
+      - add_ssh_keys:
+          fingerprints:
+            - "04:66:ee:c2:be:fb:8d:c8:ec:5d:03:ce:ab:40:88:2c"
+      - run:
+          command: |
+              git config --global user.email "ci@pyne.io"
```

Comment:
Check whether these are necessary, and if so, we should decide what the best entries are.

Reply:
it is required... what are the best entries then ?